Assets/Tests/Tests/EventTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Plugins.CountlySDK.Models;
+using Plugins.CountlySDK;
+using Plugins.CountlySDK.Enums;
+using System.Threading.Tasks;
+
+namespace Tests
+{
+    public class EventTests
+    {
+        private readonly string _serverUrl = "https://xyz.com/";
+        private readonly string _appKey = "772c091355076ead703f987fee94490";
+
+        [OneTimeSetUp]
+        public void DbNumberSetup()
+        {
+            Countly.DbNumber = 999;
+        }
+
+        [Test]
+        public void TestNullValue()
+        {
+            CountlyConfiguration configuration = new CountlyConfiguration {
+                ServerUrl = _serverUrl,
+                AppKey = _appKey,
+            };
+
+            Countly.Instance.Init(configuration);
+            Assert.AreNotEqual(Countly.Instance.Events, null);
```

Comment:
Move this assert to the configuration test class. Delete this test here afterwards.